### PR TITLE
Bump github.com/Azure/azure-sdk-for-go/sdk/azidentity

### PR DESCRIPTION
**Pull request by ganga1980**
Bumps [github.com/Azure/azure-sdk-for-go/sdk/azidentity](https://github.com/Azure/azure-sdk-for-go) from 1.5.1 to 1.6.0.
- [Release notes](https://github.com/Azure/azure-sdk-for-go/releases)
- [Changelog](https://github.com/Azure/azure-sdk-for-go/blob/main/documentation/release.md)
- [Commits](https://github.com/Azure/azure-sdk-for-go/compare/sdk/internal/v1.5.1...sdk/azcore/v1.6.0)

---
updated-dependencies:
- dependency-name: github.com/Azure/azure-sdk-for-go/sdk/azidentity dependency-type: indirect ...

### CODE REVIEW

**Review by pfrcks — DISMISSED** [review]

File: build/windows/installer/livenessprobe/livenessprobe.cpp [hunk 1]
```diff
@@ -26,7 +26,7 @@ bool IsProcessRunning(const wchar_t *const executableName)
     PROCESSENTRY32 entry;
     entry.dwSize = sizeof(PROCESSENTRY32);
 
-    const auto snapshot = CreateToolhelp32Snapshot(TH32CS_SNAPPROCESS, NULL);
+    const auto snapshot = CreateToolhelp32Snapshot(TH32CS_SNAPPROCESS, 0);
```
Comment (pfrcks) [inline build/windows/installer/livenessprobe/livenessprobe.cpp:29]: is this change needed

Comment (ganga1980) [reply build/windows/installer/livenessprobe/livenessprobe.cpp:29]: > is this change needed

yes, there were warnings in the build base. 0 is the right parameter value to pass rather than NULL though NULL is 0.

**Review by pfrcks — APPROVED** [review]
